### PR TITLE
Adding datasets and script

### PR DESCRIPTION
**Pull request by DinoDany**

### CODE REVIEW

File: Datasets/.DS_Store
Comment (eor314) [inline Datasets/.DS_Store:1]: Please remove the .DS_Store files from all directories

File: Datasets/Scripts/download_FanthomNet_img
Comment (eor314) [inline Datasets/Scripts/download_FanthomNet_img:1]: Is this a notebook? Is the file name missing an extension?

File: Datasets/Fathomnet-dataset.xlsx
Comment (eor314) [inline Datasets/Fathomnet-dataset.xlsx:1]: Can this file be converted to CSV format? If it stays in xlsx then people will need to convert it and potentially lose some data.

File: Datasets/Coco-dataset.csv
Comment (eor314) [inline Datasets/Coco-dataset.csv:1]: Just to double check, this is just the subset used for the experiments, correct? It looks that way with ~1k entries.

File: Datasets/Scripts/.DS_Store
Comment (eor314) [inline Datasets/Scripts/.DS_Store:1]: Remove.

File: Datasets/iNat-landanimals-dataset.xlsx
Comment (eor314) [inline Datasets/iNat-landanimals-dataset.xlsx:1]: Again, please save as CSV.

File: Datasets/iNat-marineanimals-dataset.xlsx
Comment (eor314) [inline Datasets/iNat-marineanimals-dataset.xlsx:1]: Please save as CSV

File: Scripts/.DS_Store
Comment (eor314) [inline Scripts/.DS_Store:1]: Please remove.

**Review by eor314 — CHANGES_REQUESTED** [review]
This looks good with a few minor changes: 

- Remove .DS_Store files from all directories. 
- Add a .gitignore to make sure files like the DS_Store are not tracked in future commits. 
- Make sure the extensions are correct for the various files. 

Once these changes are made, we can merge and make tweaks from there! We'll need to update the readme in particular to note file version, etc.

File: Datasets/.DS_Store
Comment (DinoDany) [reply Datasets/.DS_Store:1]: Done

File: Datasets/Fathomnet-dataset.xlsx
Comment (DinoDany) [reply Datasets/Fathomnet-dataset.xlsx:1]: Done

File: Datasets/Scripts/download_FanthomNet_img
Comment (DinoDany) [reply Datasets/Scripts/download_FanthomNet_img:1]: Corrected

File: Datasets/iNat-landanimals-dataset.xlsx
Comment (DinoDany) [reply Datasets/iNat-landanimals-dataset.xlsx:1]: Done

File: Scripts/.DS_Store
Comment (DinoDany) [reply Scripts/.DS_Store:1]: Done

File: Datasets/Coco-dataset.csv
Comment (DinoDany) [reply Datasets/Coco-dataset.csv:1]: Yes(:

**Comment by eor314** [comment]
@DinoDany apologies it took so long to merge this. Thanks for making the changes. Next step will be to put together a README and then I think it is done!

**Pull request merged**